doc/man/8/cephfs-shell.rst

```diff
@@ -56,7 +56,7 @@ Options
 
 .. code:: bash
 
-    [build]$ python3 -m venv venv && source venv/bin/activate && pip3 install cmd2
```

Comment:
This seems like a different fix. Although its in the docs, but does this need a separate PR?

Reply:
> This seems like a different fix. Although its in the docs, but does this need a separate PR?

This errored out without installing `colorama and packaging` while trying to start cephfs-shell. So I've updated the docs that didn't have them. Anyway, `packaging` is needed for this PR atleast.

Reply:
ack